auth/server/src/main/java/com/seagates3/authorization/Authorizer.java

```diff
@@ -79,6 +78,12 @@ ServerResponse authorize(Requestor requestor,
     ServerResponse serverResponse = null;
     AuthorizationResponseGenerator responseGenerator =
         new AuthorizationResponseGenerator();
+    // Deny access if any action is restricted for public access
+    if (requestor == null &&
+        PublicAccessAuthorizer.isActionRestricted(requestBody)) {
+      return responseGenerator.AccessDenied(
+          "Anonymous users cannot copy objects.Please authenticate.");
```

Comment:
will this error response will be given to client ? If yes, is it as per AWS error message.

We need to give space after full stop in above error response.

Reply:
Yes error text is as per AWS